commands/eval-ro.md

```diff
@@ -0,0 +1,9 @@
+This is a variant of the `EVAL` command that can not execute commands that modify keys or values. So unlike `EVAL`, scripts executed with this command will not become unkillable and will never replicate traffic. This command can also always be routed to primaries or replicas and will execute consistently on both.
```

Comment:
```suggestion
This is a read-only variant of the `EVAL` command that isn't allowed to execute commands that modify data. Unlike `EVAL`, scripts executed with this command can always be killed and never affect the replication stream. Because it can only read data, this command can always be executed on a master or a replica.
```